axum/src/lib.rs

```diff
@@ -418,7 +418,9 @@
 )]
 #![deny(unreachable_pub, private_in_public)]
 #![allow(elided_lifetimes_in_paths, clippy::type_complexity)]
-#![forbid(unsafe_code)]
+// can't be `forbid` since we've vendored code from hyper-util that contains `unsafe`
+// when hyper-util is on crates.io we can stop vendoring it and go back to `forbid`
+#![deny(unsafe_code)]
```

Comment:
Why not a git dependency?

Reply:
I'm considering doing an rc release of 0.7 when tower-http has been updated so would like to avoid git dependencies for that reason.